src/System.Net.Sockets/tests/FunctionalTests/SocketOptionNameTest.cs

```diff
@@ -163,6 +163,30 @@ public void MulticastInterface_Set_InvalidIndex_Throws()
             await MulticastInterface_Set_IPv6_Helper(0);
         }
 
+        [Fact]
+        public void MulticastTTL_Set_IPv4_Succeeds()
```

Comment:
So we were already correctly handling IPv4, just not IPv6, correct?

Reply:
yes, IPv4 works. On windows it is same call but not on linux.